patterns/1-initial/explicit-shared-ownership.md

```diff
@@ -0,0 +1,52 @@
+# Title
+
+Explicit Shared Ownership
+
+# Patlet
+
+A software component that several teams depend on has grown to the point where owners are no longer capable of taking full ownership. There is confusion who to involve for changes. Sharing ownership explicitly and making expected behaviour visible removes ambiguity. Writing a contributions document creates a natural way to evolve ownership.
```

Comment:
What do you mean by "contributions document" here?
Are you referring to a CONTRIBUTING.md, or do you not want to be that prescriptive at this point?

Reply:
This should be referring to the CONTRIBUTING.md - though it really could be written down anywhere.